renderdoc/driver/gl/gl_manager.h

```diff
@@ -92,6 +92,26 @@ class GLResourceManager : public ResourceManager<GLResource, GLResource, GLResou
     ResourceManager::Shutdown();
   }
 
+  void DeleteContext(void *context)
+  {
+    unsigned count = 0;
+    for(auto it = m_CurrentResourceIds.begin(); it != m_CurrentResourceIds.end(); it++)
+    {
+      if(it->first.Context == context)
+      {
+        ++count;
+        auto res = it->second;
```

Comment:
`auto` is fine for iterators where the type is tedious to type out, but I prefer to avoid it when the type can be made clear, like here it's just `ResourceId`.

Also ditto the other comment about `unsigned` for the `count` variable here.